lang/pt-BR.json

```diff
@@ -1929,7 +1935,8 @@
   "TierPage.ContributorsDescription": "Junte-se a nós em contribuir para este nível!",
   "TierPage.Share.title": "Ajude {collective} a alcançar sua meta!",
   "tiers.about": "Configure os níveis de contribuição",
-  "Tiers.CustomTierDescription": "Um nível padrão que permite contribuições livremente personalizáveis, para que as pessoas possam definir o valor e a frequência sem limitações. Você não pode alterar as configurações ou a descrição deste nível, mas pode desativá-lo.",
```

Comment:
I think we shouldn't discard this because of a typo fix.

Reply:
I've updated the lang files, now it should be good. 👍🏽